Sources/SmartPayment/SmartPayment.h

```diff
@@ -70,6 +76,15 @@
 // "NTA" attribute
 @property (nonatomic, strong) NSString * notificationAddress;
 
+// "FRQ" attribute
+@property (nonatomic, strong) NSString * frequency;
+
+// "DL" attribute
+@property (nonatomic, strong) NSDate * lastDate;
+
+// "DH" attribute
+@property (nonatomic, strong) NSNumber * deathHandling;
```

Comment:
I don't like `NSNumber` here and I would prefer BOOL directly, but I guess the `nil` is also the option (e.g. if the field is not in the SPAYD, then it has also some meaning, right?)